include/mata/nfa/builder.hh

```diff
@@ -7,13 +7,17 @@
 
 #include <filesystem>
 
-using namespace Mata::Nfa;
 
 /**
  * Namespace providing options to build NFAs.
  */
 namespace Mata::Nfa::Builder {
 
+using namespace Mata::Nfa;
+
+using StateNameValueMap = std::unordered_map<std::string, State>;
```

Comment:
Just NameStateMap ? A map from names to states.

Reply:
It was meant to be read as “state name to state value map”. But I agree that the name is not great. It could be shortened to a generic “name to state map”. It is still clear enough to understand.

Reply:
This issue created for this problem: https://github.com/VeriFIT/mata/issues/291.

Reply:
yes please :), StateNameMap, without the "to"

Reply:
Done.